examples/testPromiseGetNextPage.js

```diff
@@ -1,6 +1,6 @@
 const GitHubApi = require('github')
 
-var gh = new GitHubApi();
+var gh = new GitHubApi()
```

Comment:
can you also pass `{debug: true}` and rename `gh` to `github` to follow the other examples?